include/seqan3/alignment/pairwise/align_pairwise.hpp

```diff
@@ -66,17 +63,17 @@ namespace seqan3
  *
  * In many cases one needs to compute multiple pairwise alignments. Accordingly, the align_pairwise interface allows
  * to pass a range over sequence pairs. The alignment algorithm will be configured only once for all submitted
- * alignments and then computes the alignments sequentially or in parallel depending on the given execution policy.
+ * alignments and then computes the alignments sequentially or in parallel depending on the given configuration.
  * Since there is always a certain amount of initial setup involving runtime checks required, it is advisable to pass
  * many sequence-pairs to this algorithm instead of repeatedly calling it with a single pair.
  *
  * ### Accessing the alignment results
  *
  * For each sequence pair one or more \ref seqan3::alignment_result "seqan3::alignment_result"s can be computed.
  * The seqan3::align_pairwise function returns an seqan3::alignment_range which can be used to iterate over the
- * alignments. If the sequenced execution policy is used the alignments are computed on-demand when iterating over the
- * results. In case of a parallel execution all alignments are computed at once in parallel when calling `begin`
- * on the associated seqan3::alignment_range.
+ * alignments. If the `parallel` and `vectorise` configurations are omitted the alignments are computed on-demand when
```

Comment:
The vectorised alignment does not cache all results at once. Only the parallel mode would do.